frontend/src/components/Trial/Trial.tsx

```diff
@@ -17,7 +17,7 @@ export interface IFeedbackForm {
     is_skippable: boolean;
 }
 
-interface TrialProps {
+export interface TrialProps {
     playback: any;
```

Comment:
We should still type the playback arguments as well (can be left for later).

Reply:
Good spot. It might be possible to use the `PlaybackArgs` type that we already have. I will investigate it.